cmd/bootstrap/main.go

```diff
@@ -221,14 +230,28 @@ func getTopology(peers sets.String) (string, []string, error) {
 			return "", nil, errors.Wrap(err, "check replication status")
 		}
 
-		if status == mysql.ReplicationStatusActive {
-			replicas.Insert(peer)
+		replicaHost, err := db.ReportHost()
+		if err != nil {
+			return "", nil, errors.Wrap(err, "get report_host")
+		}
+
+		if replicaHost != "" && peers.Len() > 1 {
+			replicas.Insert(replicaHost)
+		}
+
+		if status == mysql.ReplicationStatusActive && replicaHost != "" {
```

Comment:
how about?

```suggestion
		if replicaHost == "" {
			continue
		}

		if peers.Len() > 1 {
			replicas.Insert(replicaHost)
		}

		if status == mysql.ReplicationStatusActive {
```

Reply:
yes, thank you I forgot to remove unneeded checks